Yoast/wordpress-seo
# Remove about notice

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Look for the notice and remove it when it does exists.

## Test instructions

This PR can be tested by following these steps:

* Check if the notice has been removed.

Fixes #5862
